source/builders.txt

```diff
@@ -0,0 +1,305 @@
+.. _php-builders:
+
+========================
+Operations with Builders
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: aggregation, query, code example, type-safe
+
+Overview
+--------
+
+In this guide, you can learn about the **builder classes**
+that the {+library-short+} provides to create types used in your
+operations. You can use the builder classes and factory methods from the
+:ref:`php-aggregation-builder-api` feature to create filters for other
+operations such as find, update, and delete operations.
+
+Using builders to create queries helps you identify errors at compile
+time and avoid them at runtime. This guide provides information on
+builder classes that you can use for the following tasks:
```

Comment:
```suggestion
time and avoid them at runtime. This guide provides information on
builder classes in the following sections:
```
S: Introduce that these are links to the below sections. Consider using the same names as the section titles as well.